image/update-pong.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Is this script really needed? It only extracts the file, right? You seem to extract the file in go code, too, so it seems redundant? (If you remove it, don't forget to remove the reference in the Makefile)